src/collection.coffee

```diff
@@ -125,7 +126,7 @@ module.exports = class Collection extends Backbone.Collection
     @loaded = state
     @trigger 'loaded', this if state && options.trigger
 
-  update: (models) ->
+  update: (models, silent = false) ->
```

Comment:
It would be sweet to just future proof this:

```suggestion
  update: (models, opts = { silent: false }) ->
```

I dunno if the syntax is good but I think I get the point across.

Reply:
Good call. Done! :+1: